Lib/pydoc.py

```diff
@@ -587,6 +577,35 @@ def getdocloc(self, object, basedir=sysconfig.get_path('stdlib')):
             docloc = None
         return docloc
 
+    def _get_version(self, object):
+        if self._is_stdlib_module(object):
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore", DeprecationWarning)
+                return str(getattr(object, '__version__', None))
+        else:
+            return str(getattr(object, '__version__', None))
```

Comment:
```suggestion
        if self._is_stdlib_module(object):
            with warnings.catch_warnings():
                warnings.simplefilter("ignore", DeprecationWarning)
                version = getattr(object, '__version__', None)
        else:
            version = getattr(object, '__version__', None)
        return '' if version is None else str(version)
```

We don't a non-existing version to be printed as `version: None` I think.